src/websocket.ts

```diff
@@ -57,9 +54,8 @@ export default class WS {
 
   on(
     eventName: "connection" | "message" | "close",
-    callback: (socket: Server) => void
+    callback: (socket: WebSocket) => void
```

Comment:
I think that `Server` was the correct type? This change seems to be breaking the type checking here: https://github.com/romgain/jest-websocket-mock/blob/master/src/__tests__/websocket.test.ts#L145

See the typescript logs in Travis here: https://travis-ci.org/romgain/jest-websocket-mock/jobs/630849316#L261

Reply:
Thank you for review. In the latest I've reverted `ts-ignore` for this function. Do you think it is good?

Reply:
Thanks! 

Unfortunately, I think that's the best we can do for now 🙂